posts/job_descriptions.md

```diff
@@ -0,0 +1,86 @@
+engineering staff when we're recruiting. We're not Google or Apple, with a huge
+global presence, most developers have probably never heard of us. The current
+job market for developers is really competitive, there aren't that many great
+developers out there, most of them are already in work and everyone is competing
+for them.
+
+We've recently been recruiting for an iOS developer role, to lead the development
+of our telematics apps. I think it's actually a really interesting, fun role, which
+will involve lots of really quite difficult tasks, really stretching the technology.
+
+Our challenge is to make sure that we can get that across in our job adverts.
+
+Our process includes a couple of [coding tests](https://github.com/mydrive/code-tests). One of which
+candidates can do on their own at home, and a much smaller one which we do in
+house, just to verify what we've seen on the external test.
+
+A few candidates have responded and taken the test and it gives a
+good view of where someone is in their development. It also gives you a real
+example to discuss with a candidate about why they made certain decisions, what
+their thought processes were, and what they'd do differently next time. We haven't
+found the right person yet, but it's been really worthwhile.
+
+Some recruiters have told us that people are put off from applying because there
+is a more challenging process including a test, and hopefully that's just filtering
+out people who aren't suitable for the role, but we could be putting off some
+candidates who really would be great because we've increased the barrier to apply.
+
+All of this caused us to review our Job Descriptions. We quickly realised
+that just tweaking wording maybe wasn't radical enough. There was also a feeling
+that one of the best things about working at [@MyDrive](https://twitter.com/_mydrive) is the working culture
+which wasn't well reflected in the job descriptions.
+
+We have a lot of good developers, what would we want to see in a job advert?
+How do we get across what a good place it is to work?
+
+We had a small brainstorming session, splitting things into two sections:
+
+1. What we will do for the candidate?
+2. What they will do for us?
+
+![Categories](/assets/media/job_voting/categories.jpg)
+
+
+We got two great lists, and decided to vote on which of the things we'd got on
+our list were the most important, so we could focus on those in our adverts.
+
+![Results](/assets/media/job_voting/results.jpg)
+
+The biggest winner was Remote Working Encouraged, which is really interesting. My
+offer letter contained the phrase "some remote working encouraged", and I remember
+thinking at the time that that statement was great. It's not described as allowed, 
+or merely tolerated at MyDrive it's encouraged. There is an understanding that some
+remote working is actually beneficial, both to individuals and to the company, and
+the tools are in place to make remote working easy. 
```

Comment:
I'd like to rephrase this paragraph. I don't know that "encouraged" is the right verb, "actively supported" might be more appropriate; working from home is subject to management consent, and isn't a contractual right. So long as it doesn't negatively impact the business (indeed it usually positively impacts) then we'll be extremely accommodating compared with most companies.